CMakeLists.txt

```diff
@@ -100,7 +102,12 @@ CHECK_INCLUDE_FILES("sys/types.h" HAVE_SYS_TYPES_H)
 CHECK_INCLUDE_FILES("sys/vfs.h" HAVE_SYS_VFS_H)
 CHECK_INCLUDE_FILES("sys/prctl.h" HAVE_SYS_PRCTL_H)
 CHECK_INCLUDE_FILES("execinfo.h" HAVE_EXECINFO_H)
+if(${LINUX})
```

Comment:
On 30-6-2016 11:38, Kefu Chai wrote:
> In CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10024#discussion_r69102430>:
> 
>> @@ -100,7 +102,12 @@ CHECK_INCLUDE_FILES("sys/types.h" HAVE_SYS_TYPES_H)
>>  CHECK_INCLUDE_FILES("sys/vfs.h" HAVE_SYS_VFS_H)
>>  CHECK_INCLUDE_FILES("sys/prctl.h" HAVE_SYS_PRCTL_H)
>>  CHECK_INCLUDE_FILES("execinfo.h" HAVE_EXECINFO_H)
>> +if(${LINUX})
> 
> |if(LINUX)| would suffice.

I see both variations (I think), but if this is mode of preference
I'll fix all cases of this.